### DISCUSSION TITLE
submit error

### DISCUSSION THREAD

**Discussion by billbiancheng** [PaddlePaddle/PaddleSpatial#132] [BaiduKDDCup2022] [answered]
hello
The results submitted two days ago already reflect the score, but today it shows an error. thank you 
![image](https://user-images.githubusercontent.com/26194579/171339965-6c348062-dcb5-4c40-b1c4-170c57fdec38.png)

**Comment by ramber1836** [comment, marked as answer]
This error means that you should change the value of 'framework' in the preprare.py of your submitted code to 'tensorflow'.
